fix (timeline block): align dot if background is enabled in mobile

mxkae opened the pull request:
fixes #2863

bfintal commented on src/block/timeline/style.scss:
```diff
@@ -125,9 +125,6 @@
 	.stk-block-timeline > .stk-inner-blocks::after {
 		inset-inline-start: calc(var(--line-dot-size, 16px) / 2 - var(--line-bg-width, 3px) / 2 + 16px);
 	}
-	.stk-block-timeline.stk-block-background .stk-block-timeline__middle {
-		margin-left: -16px;
-	}
```
This might cause previously existing timeline blocks to break in mobile.

bfintal reviewed (changes requested):
Ideally, the fix should not alter existing timeline blocks (unless they're broken and this fixes it). But from the looks of the code, the removal of the style in `style/scss` which is fixed by the generated style in `style.js` might break pages with already existing timeline blocks.

mxkae commented:
added timeline blocks (branch w/o fix):

https://github.com/gambitph/Stackable/assets/98727316/ea800d61-8426-4742-9f6c-7f4b358f9eba

switched to branch w/ fix:

https://github.com/gambitph/Stackable/assets/98727316/fc444641-102a-4b5f-8d81-90fe53246aa8
